REFERENCE.md

```diff
@@ -68,14 +66,17 @@ interface QueueOptions {
 }
 ```
 
+When a job gets hit with the rate limiter, it is moved to the delayed queue.
+
 ```typescript
 interface RateLimiter {
-  max: number,      // Max number of jobs processed
-  duration: number, // per duration in milliseconds
+  max: number; // Max number of jobs processed
+  duration: number; // per duration in milliseconds
+  discard: boolean = false; // When jobs get rate limited, nothing happens
```

Comment:
`discard` might not be the best name for this.. because we're not entirely discarding the rate limiter, we are just disabling the delayed queue. So maybe `disableDelyedQueue` or something similar?

Reply:
ok, now I full understand the use-case and discard may indeed not be the best name. I thought the idea was to really throw away the jobs that get "throttled". so this name will be misleading. On the other hand, a name like ```disableDelayedQeuue``` is a bit techie and exposes implementation details.

Reply:
Not sure about this name but what about this alternative ```bounceBack``` ? the option is quite special since it only has use in the manual processing pattern...

Reply:
I like `bounceBack` 👍